Tests/swift-sdk-swift-tests/DeprecatedFunctionsTests.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  DeprecatedFunctionsTests.swift
+//  swift-sdk-swift-tests
+//
```

Comment:
Not needed for +1 but I usually remove the above two lines when I add new files. Sometimes we need to change file names when refactoring and this becomes stale. Besides, this doesn't really add any information. Anyway, this is just a comment for future commits.